src/quickgui/plugin/qgsquickmapcanvas.qml

```diff
@@ -0,0 +1,142 @@
+
+/***************************************************************************
+ qgsquickmapcanvas.qml
+  --------------------------------------
+  Date                 : 10.12.2014
+  Copyright            : (C) 2014 by Matthias Kuhn
+  Email                : matthias@opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+import QtQuick 2.3
+import QtQuick.Controls 2.2
+import QtQml 2.2
+import QgisQuick 0.1 as QgsQuick
```

Comment:
What is the reason for this alias? Couldn't that lead to confusion when people start googling for one or the other?

Reply:
good point, renamed to QgsQuick